### DISCUSSION TITLE
I want to participate in GSoC 2023, need help to get started with open source.

### DISCUSSION THREAD

**Discussion by dmdbilal** [kiwix/kiwix-android#3195] [Q&A] [answered]
Hey everyone, I am Mohamed Bilal , second year B.Tech student at Sri Sairam engineering college , Chennai, India. I know Android development related stuffs : Kotlin , MVVM, Room, APIs , Retrofit etc.. I need guidance to start working on this project.So, can you tell me more about this and guide me further about the same.

**Comment by gouri-panda** [comment, marked as answer]
Hi @dmdbilal ! , entry point is [kiwix.org/code](http://kiwix.org/code)

**Reply by dmdbilal** [reply]
> Hi @dmdbilal ! , entry point is [kiwix.org/code](http://kiwix.org/code)

@gouri-panda Does kiwix Android participating in GSoC ?
